code/modules/projectiles/guns/hookshot.dm

```diff
@@ -506,3 +506,140 @@
 	icon_state = "vampkiller_true"
 	hooktype = /obj/item/projectile/hookshot/whip/vampkiller/true
 
+//Windup-Boxes/////////////////////////////////////////////////////
+
+/obj/item/weapon/gun/hookshot/whip/bootbox
+	name = "boot-in-a-box"
+	icon = 'icons/obj/wind_up.dmi'
+	icon_state = "bootbox-0"
+	item_state = "bootbox"
+	desc = "A box with a spring-loaded boot inside. There is a crank attached to wind it up."
+	fire_action = "activate"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guns_experimental.dmi', "right_hand" = 'icons/mob/in-hand/right/guns_experimental.dmi')
+	clumsy_check = 0 //Just makes sense
+	force = 5
+	maxlength = 4 //whips are weird and this ends up being 2
+	hooktype = /obj/item/projectile/hookshot/whip/bootbox
+	var/windUp = 0 //amount of times cranked
+	var/maxWindUp = 16 //threshold for overwind
+	var/overWind = 0 //warning/delay system
+	var/state = 0 //Icon changes with each crank, stolen from the crank cell charger
+	var/springForce = 0 //makes the big kicks
+
+/obj/item/weapon/gun/hookshot/whip/bootbox/update_icon()
+	icon_state = "bootbox[state ? "-1" : "-0"]"
+
+/obj/item/weapon/gun/hookshot/whip/bootbox/attack_self(mob/user)
```

Comment:
needs sanity, make sure they're not incapacitated so that you can't wind the bootbox if you died while buckled into a chair